config/general.go

```diff
@@ -0,0 +1,51 @@
+// Copyright © 2022 Meroxa, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package config
+
+import (
+	"github.com/conduitio/conduit-connector-oracle/config/validator"
+	"github.com/conduitio/conduit-connector-oracle/models"
+)
+
+// A General represents a general configuration needed to connect to Oracle database.
+type General struct {
+	// Username is the configuration of the username to connect to Oracle database.
+	Username string `json:"username" validate:"required,lte=128,oracle"`
+
+	// Password is the configuration of the password to connect to Oracle database.
+	Password string `json:"password" validate:"required,lte=30,oracle"`
+
+	// URL is the configuration of the path to the Oracle database.
+	URL string `json:"url" validate:"required"`
+
+	// Table is the configuration of the table.
+	Table string `json:"table" validate:"required,lte=128,oracle"`
+}
```

Comment:
Where we can schema?

Reply:
```In Oracle, users and schemas are essentially the same things. You can consider that a user is an account you use to connect to a database, and a schema is the set of objects (tables, views, etc.) that belong to that account.```
https://dba.stackexchange.com/a/37022

You can see a flow to create a new schema here:
https://docs.oracle.com/en/cloud/paas/exadata-express-cloud/csdbp/create-database-schemas.html#GUID-C20C60A5-A655-4CAA-957A-93BF28F4E5FA

So I don't think it's necessary to add an additional schema field.